src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/CNNScoreVariants.java

```diff
@@ -108,6 +111,9 @@
             "1D models will look at the reference sequence and variant annotations." +
             "2D models look at aligned reads, reference sequence, and variant annotations." +
             "2D models require a BAM file as input as well as the tensor-type argument to be set.";
+    static final String AVXREQUIRED_ERROR = "This tool requires AVX instruction set support by default due to its dependency on recent versions of the TensorFlow library.\n" +
+            " If you have an older (pre-1.6) version of TensorFlow installed that does not require AVX you may attempt to re-run the tool with the --disable-avx-check argument to bypass this check.\n" +
+            " Note that such configurations are not officially supported.";
```

Comment:
Please add a string constant `DISABLE_AVX_CHECK_NAME` for the value `disable-avx-check`, and replace the argument name  thats embedded in this String with a `%s` format specifier. When the exception is thrown below, call call String.format passing it this string and the `DISABLE_AVX_CHECK_NAME` string.